src/ucp/core/ucp_request.h

```diff
@@ -300,7 +307,9 @@ struct ucp_request {
             uct_tag_context_t     uct_ctx;  /* Transport offload context */
             ssize_t               remaining;  /* How much more data
                                                * to be received */
-            ucs_ptr_map_key_t     rreq_id;  /* the receive request ID on receiver side */
+
+            /* Remote request ID received from a peer */
+            ucs_ptr_map_key_t     remote_req_id;
```

Comment:
why need to rename other protocols' fields to "remote_req_id" in this PR?

Reply:
we decided to make it more clear - instead of just `req_id` or etc

Reply:
yes, but i think we wanted to push it to separate independent PR?

Reply:
ok, I will separate unrelated to refactoring changes to separate PR

Reply:
didn't we move this to separate PR?

Reply:
no, we leave it as local ID there.